src/OrbitCore/Path.cpp

```diff
@@ -4,11 +4,20 @@
 
 #include "Path.h"
 
+#ifdef _WIN32
+#define getpid GetCurrentProcessId
```

Comment:
this is not a good practice, Maybe orbit_base::GetCurrentProcessId() which is cross-platform and calls correct implementation. Like GetCurrentThreadId in OrbitBase ThreadUtils.h

Reply:
Done. Thanks for the suggestion!